content/docs/instrumenting/exporters.md

```diff
@@ -65,6 +65,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NVIDIA GPU exporter](https://github.com/mindprince/nvidia_gpu_prometheus_exporter)
    * [ProSAFE exporter](https://github.com/dalance/prosafe_exporter)
    * [Ubiquiti UniFi exporter](https://github.com/mdlayher/unifi_exporter)
+   * [Netgear router exporter](https://github.com/DRuggeri/netgear_exporter)
```

Comment:
Keep it sorted

Reply:
Roger that - fixed in 48ec9debdce68db10c7f297a410f0c4eb84b8e89